content/strategy-goals/strategy/code-intelligence/index.md

```diff
@@ -79,22 +79,21 @@ Top feedback patterns we're hearing from customers and prospects across the boar
-- Accelerating [Developer onboarding](../use-cases/dev-onboarding.md) by enabling the ability to search over the full transitive dependency graph.
-- Monitor [Code Health](../use-cases/code-health.md) by creating Code Insights that track deprecated dependency usages and version drift.
-- Encourage [Code Reuse](../use-cases/code-reuse.md) by making precise navigation the obvious way to find references to relevant usage examples.
-
-These are all currently in early stages of development and discussion. Read more about how we're thinking about the dependency graph in [RFC 593: Unlocking use cases based on the dependency graph of repositories](https://docs.google.com/document/d/1SkM8CG0IksvPEKRBRVLKipiRJTopx6Vq_hSWRJ9NyKs/edit#heading=h.trqab8y0kufp).
+Several recent efforts have shown an increasing need to unify the [code intelligence data platform](https://docs.google.com/document/d/1AjZ_d0nJVHbV75IH3jZRkrGXhsv_AXp2kS4nrw2SAQ8). These efforts show an immediate need to be able to correlate and aggregate data from different sources within an API boundary (instead of re-implemented on several clients). This will set precedent for where code goes in relation to code intelligence domains and will establish clear ownership boundaries for other product teams to build on top of. We see this engineering milestone as a large enabler for every product vertical, and a key part in enabling deep integration among our code-related concepts as a code intelligence platform.
 
 **Ship precise language support**
 
-We’ve historically invested in broadening our span of supported languages. This is an ongoing effort that ties directly back to the Global Code Graph vision, aiming to support languages that cover 90% of code intelligence actions at customers and Sourcegraph Cloud (see our [language share dashboard section](https://sourcegraph.looker.com/dashboards/159)). Increasing precise support contributes to push forward any security features we choose to implement as it makes compiler-accurate metadata available for all our product verticals. We're currently working on adding precise cross-repository Python support and PyPi package support.
+We’ve historically invested in broadening our span of supported languages. This is an ongoing effort that ties directly back to the Global Code Graph vision, aiming to support languages that cover 90% of code intelligence actions at customers and Sourcegraph Cloud (see our [language share dashboard section](https://sourcegraph.looker.com/dashboards/159)). Increasing precise support contributes to push forward any security features we choose to implement as it makes compiler-accurate metadata available for all our product verticals. We're currently working on:
+
+- Adding precise cross-repository Python and PyPi package support in preparation for our Python indexer launch.
```

Comment:
Do we want to add that outside of Ruby/C++ we're unlikely to add any more precise languages _based_ on current demand/team headcount?